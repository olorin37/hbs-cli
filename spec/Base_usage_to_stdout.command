#!/usr/bin/env bash
exec "$command_under_test" data.yaml standalone.hbs > "$output_file"
