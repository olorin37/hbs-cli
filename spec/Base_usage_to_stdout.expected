Number: 1234567899.
Pi: 3.14.
Hello, I am hbs.
